pkg/resources/procedure.go

```diff
@@ -124,6 +130,7 @@ var procedureSchema = map[string]*schema.Schema{
 		ForceNew:     true,
 		ValidateFunc: validation.StringInSlice([]string{"VOLATILE", "IMMUTABLE"}, false),
 		Description:  "Specifies the behavior of the function when returning results",
+		Deprecated:   "These keywords are deprecated for stored procedures. These keywords are not intended to apply to stored procedures. In a future release, these keywords will be removed from the documentation.",
```

Comment:
This addresses https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2291? Can you please add this to PR description (Fix #xxxx)?

Reply:
added